docs/sites.yml

```diff
@@ -5372,3 +5372,12 @@
     - Portfolio
     - Personal Website
     - Photography
+- title: Pantene
+  main_url: https://pantene.com
+  url: https://pantene.com
+  featured: false
+  description: >
+    Pantene is a Swiss-created American brand of hair care products owned by Procter & Gamble
+  categories:
+    - Product Website
+    - Hair care product website
```

Comment:
It would be great if we could pick one of already existing categories and not create new ones that would contain single entry

My suggestion would be:
 - Beauty
 - Business

Reply:
ok I will make changes in the file and create a new PR